File: example/javalib/hello-world/java/com/helloworld/app/MainActivity.java
package com.helloworld.app;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import android.view.ViewGroup.LayoutParams;
import android.view.Gravity;


public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: example/kotlinlib/android/1-hello-world/app/src/main/kotlin/com/helloworld/app/MainActivity.kt
package com.helloworld.app

import android.app.Activity
import android.os.Bundle
import android.view.Gravity
import android.view.ViewGroup.LayoutParams
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Create a new TextView
        val textView = TextView(this)

        // Set the text to "Hello, World!"
        textView.text = "Hello, World Kotlin!"

        // Set text size
        textView.textSize = 32f

        // Center the text within the view
        textView.gravity = Gravity.CENTER

        // Set layout parameters (width and height)
        textView.layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT)

        // Set the content view to display the TextView
        setContentView(textView)
    }
}
